DataEngineering/fetch_data.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+import pandas as pd
+import json
+import requests
+import mysql.connector as mysql
+
+url = "https://api.tfl.gov.uk/bikepoint" 
+response = requests.get(url)
+data = pd.DataFrame(json.loads(response.text))
+
+data['time'] = datetime.now()
+data.drop(['url','$type','placeType','children','childrenUrls'],axis=1,inplace=True)
+
+data['NbBikes'] = data['additionalProperties'].apply(lambda x : x[6]['value'])
+data.drop(['additionalProperties'],axis=1,inplace=True)
+
+db = mysql.connect(user='root', password='',host='localhost', database='db')
+cursor = db.cursor()
+
+for ele in data.iterrows():
```

Comment:
A cleaner approach when you're not using the index from the (index, row) tuple is to write it like this:
`for _ , row in data.iterrows()`. That ways, you don't have to do the extra step - `row = ele[1]`